src/main/java/com/senlainc/warsaw/tyurin/util/propertyHandlers/PropertyReader.java

```diff
@@ -0,0 +1,38 @@
+package com.senlainc.warsaw.tyurin.util.propertyHandlers;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+
+    private static PropertyReader INSTANCE;
+    private Properties properties;
+
+    private PropertyReader() {
+        properties = new Properties();
+    }
+
+    public static PropertyReader getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new PropertyReader();
+        }
+        return INSTANCE;
+    }
+
+    public String readProperties(String propertyName, String path) {
+        try (InputStream input = new FileInputStream(path)) {
```

Comment:
`InputStream input = PropertyReader.class.getClassLoader().getResourceAsStream(path)`

You can do it like this (check it by yourself, i didn't tested it)